apps/opik-backend/src/test/java/com/comet/opik/api/resources/utils/TestDropwizardAppExtensionUtils.java

```diff
@@ -38,7 +38,8 @@ public record AppContextConfig(
             String awsJdbcDriverPlugins,
             boolean usageReportEnabled,
             String usageReportUrl,
-            String metadataVersion) {
+            String metadataVersion,
+            boolean corsEnabled) {
```

Comment:
Minor: these overloads aren't going to be sustainable, I'd go directly with the `AppContextConfig` object by using its builder.

Reply:
Done in https://github.com/comet-ml/opik/pull/498/commits/bda561947b6abf3533aed111e765066bbd918ca4